backend/accounts.go

```diff
@@ -416,6 +416,12 @@ func (backend *Backend) createAndAddAccount(
 			backend.arguments.GapLimits(),
 			backend.log,
 		)
+
+		if specificCoin.Code() == coinpkg.CodeBTC ||
+			specificCoin.Code() == coinpkg.CodeTBTC {
+			specificCoin.SetFormatUnit(backend.config.AppConfig().Backend.BtcUnit)
+		}
```

Comment:
this is the wrong place for this code. first, you wouldn't expect a function that  creates an account to modify the underlying coin (which is shared between multiple accounts). secondly, this is redundant, as it will be called for the same coin multiple times if there are multiple btc accounts.

maybe `backend.Coin()` is the right function to put this in, which is when the coin instance is created?

Reply:
Good point, thx. Changed